test/HotKeys/DeleteAndBackSpaceAliases.spec.js

```diff
@@ -1,45 +1,73 @@
 import React from 'react';
-import {mount} from 'enzyme';
-import {expect} from 'chai';
+import { mount } from 'enzyme';
+import { expect } from 'chai';
 import sinon from 'sinon';
 
-import {HotKeys} from '../../src/';
+import { HotKeys } from '../../src/';
 import FocusableElement from '../support/FocusableElement';
 
-const keyAliases = ['del', 'backspace', 'Delete', 'BackSpace' ];
-const keyNames = [ 'Delete', 'Backspace' ];
+['backspace', 'BackSpace'].forEach(function(keyAlias) {
+  describe(`When an action is defined using the combination '${keyAlias}':`, () => {
+    context(`and a BackSpace event is triggered`, () => {
```

Comment:
⬜️ Remove the use of backticks for this string - it no longer needs interpolation.